src/js/api/config.js

```diff
@@ -84,6 +85,11 @@ const Config = function(options, persisted) {
     config.language = language;
     config.intl = intl;
 
+    // If autoPause is configured with an empty block,
+    // default autoPause.viewability to true.
+    let autoPause = config.autoPause;
+    autoPause.viewability = ('viewability' in autoPause) ? !!autoPause.viewability : true;
```

Comment:
Now `config.autoPause` will always be defined because you added it to `Defaults`. So for this to work, you want to check what was passed into setup which is `allOptions`:

```js
let autoPause = allOptions.autoPause; // set by user
config.autoPause.viewability = ('viewability' in autoPause) ? !!autoPause.viewability : true;
```